lib/pay/paddle_billing/billable.rb

```diff
@@ -33,7 +33,13 @@ def customer
           sc
         end
       rescue ::Paddle::Error => e
-        raise Pay::PaddleBilling::Error, e
+        sc = Paddle::Customer.list(email:).data.first
+        if sc.present?
+          pay_customer.update!(processor_id: sc.id)
```

Comment:
I'm not quite sure this logic makes sense. We would end up with Pay::Customer records with duplicate `processor_id` here, so we still need to raise an error here.